CHANGELOG.md

```diff
@@ -1,29 +1,82 @@
 # posh-git Release History
 
-## 1.0.0-beta1 - January 10, 2018
+## 1.0.0-beta2 - May 13, 2018
 
 The 1.0.0 release is targeted specifically at Windows PowerShell 5.x and (cross-platform) PowerShell Core 6.x, both of
-which support writing prompt strings using [ANSI escape sequences](https://en.wikipedia.org/wiki/ANSI_escape_code).
-On Windows, support for [Console Virtual Terminal Sequences](https://docs.microsoft.com/en-us/windows/console/console-virtual-terminal-sequences)
-was added in Windows 10 version 1511.
-Consequently this release introduces BREAKING changes with 0.7.x by:
-
-- Dropping support for Windows PowerShell versions 2.0, 3.0 and 4.0.
-- Changing the $GitPromptSettings hashtable to a more structured and stongly typed object.
-  Here is one example of the changed settings structure:
-  ```powershell
-  $GitPromptSettings.LocalWorkingStatusSymbol = '#'
-  $GitPromptSettings.LocalWorkingStatusForegroundColor = [ConsoleColor]::DarkRed
-  ```
-  Changes to:
-  ```powershell
-  $GitPromptSettings.LocalWorkingStatusSymbol.Text = '#'
-  $GitPromptSettings.LocalWorkingStatusSymbol.ForegroundColor = [ConsoleColor]::DarkRed
-  ```
-- Changing `Write-VcsStatus`, `Write-GitStatus` and `Write-Prompt` to return a string rather than write to host when
-  the host supports ANSI escape sequences.
+which support classes, enabling the enhanced structure of `$GitPromptSettings`,
+and writing prompt strings using [ANSI escape sequences](ansi-esc-code) /
```

Comment:
Change this `[ANSI escape sequences](ansi-esc-code)` to this `[ANSI escape sequences][ansi-esc-code]` and it should work.  Ditto for the other two.

Reply:
Sigh. Thanks.

Any other feedback?

Reply:
Other than those three links, it looks good!